File: Prime Checker/prime_checker.js
function isPrime(number) {
  //to be a prime it must be divided only by himself or by 1

  //1 is not a prime
  if (number == 1) {
    return false;
  }
  else {
    for (let i = 2; i < number; i++) {
      if (number % i == 0){
        return false;
      }
    }
    return true;
  }
}

//making tests

console.log(isPrime(2));
//true

console.log(isPrime(4));
//false

console.log(isPrime(97));
//true

console.log(isPrime(98));
//false

File: Prime Checker/prime_checker.py
def isPrime(number):
    ##to be a prime it must be divided only by himself or by 1
    
    ##1 is not a prime, neither 0 or negative numbers
    if (number <= 1):
        return False;
    else:
        for i in range(2, number):
            if (number % i == 0):
                return False;
      
    return True;